ts/Accessibility/Components/SeriesComponent/SeriesDescriber.ts

```diff
@@ -63,6 +63,12 @@ const {
  *
  * */
 
+declare module '../../../Core/Series/SeriesOptions' {
+    interface SeriesOptions {
+        nullInteraction?: boolean;
+    }
+}
+
```

Comment:
`nullInteraction` is declared in multiple files - here and in `ts/Accessibility/Components/SeriesComponent/SeriesKeyboardNavigation.ts`, `ts/Core/Pointer.ts`, `ts/Core/Series/DataLabel.ts`, `ts/Core/Series/Series.ts` is it necessary? Can we avoid it and declare it in one place? I think it should be declared once for Series.

Reply:
I am not entirely sure about this part :grimacing:

Option is inteded for LineSeries and series which extend it. Any ideas on how best to declare it? 🤔